Language/Functions/Math/abs.adoc

```diff
@@ -45,7 +45,34 @@ Calculates the absolute value of a number.
 // HOW TO USE SECTION STARTS
 [#howtouse]
 --
+[float]
+=== Example Code
+// Describe what the example code is all about and add relevant code   ►►►►► THIS SECTION IS MANDATORY ◄◄◄◄◄
+Prints the absolute value of variable a to the Serial Monitor.
```

Comment:
```suggestion
Prints the absolute value of variable `x` to the Serial Monitor.
```
- Add [backticks](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#text-formatting) to visually distinguish the code in the sentence from the prose ("a" vs "`a`").
- Change the variable name to match the name used in the code.
   If you prefer the variable name `a` instead of `x`, feel free to change the code instead of this sentence. I don't have  a strong preference either way. I just chose `x` arbitrarily for the code I used to illustrate my previous suggestion.